doc/input-output-reference/src/overview/group-plant-equipment.tex

```diff
@@ -3860,86 +3860,68 @@ \subsubsection{Inputs}\label{inputs-10-013}
+\begin{align}
+\mathrm{Linear} \to Eff ={}& {A_0} + {A_1}\cdot PLR\\
+\mathrm{Quadratic} \to Eff ={}& {A_0} + {A_1} \cdot PLR + {A_2}\cdot PL{R^2}\\
+\mathrm{Cubic} \to Eff ={}& {A_0} + {A_1}\cdot PLR + {A_2}\cdot PL{R^2} + {A_3}\cdot PL{R^3}\\
+\mathrm{BiQuadratic} \to Eff ={}& {A_0} + {A_1}\cdot PLR + {A_2}\cdot PL{R^2} + {A_3}\cdot {T_w} + {A_4}\cdot {T_w}^2 + {A_5}\cdot PLR\cdot {T_w}\\
+\mathrm{QuadraticLinear} \to Eff ={}& {A_0} + {A_1}\cdot PLR + {A_2}\cdot PL{R^2} + {A_3}\cdot {T_w} + {A_4}\cdot PLR\cdot {T_w} + {A_5}\cdot PL{R^2}\cdot {T_w}\\
+\begin{split}
+\mathrm{Bicubic} \to Eff ={}& A_0 + A_1 \cdot PLR + A_2 \cdot PLR^2 + A_3 \cdot T_w + A_4 T_w^2 + A_5 \cdot PLR \cdot T_w + A_6 \cdot PLR^3 +\\
+&A_7 \cdot T_w^3 + A_8 \cdot PLR^2 \cdot T_w + A_9 \cdot PLR \cdot T_w^2
+\end{split}
+\end{align}
+\noindent where
+\begin{description}[labelwidth=1cm, leftmargin=!]
+\item[$Eff$] is normalized boiler efficiency.
+\item[$PLR$] is boiler part-load ratio.
+\item[$T_w$] is boiler water temperature in Celcius, this can be either entering or leaving temperature depending on the setting in the previous field.
+\end{description}
 
 \paragraph{Field: Design Water Outlet Temperature}\label{field-design-water-outlet-temperature}
 
 This numeric field contains the desired boiler water outlet temperature in Celsius.
 
 \paragraph{Field: Design Water Flow Rate}\label{field-design-water-flow-rate-001}
 
-This numeric field contains the maximum design water volumetric flow rate in m\(^{3}\)/sec.~This should be the largest flow rate than can be heated to the design outlet temperature. This field is autosizable.
+This numeric field contains the maximum design water volumetric flow rate in \si{\meter\cubic\per\second}. This should be the largest flow rate than can be heated to the design outlet temperature. This field is autosizable.
```

Comment:
Hmm, in the pdf this rendered wrong. Just happened to note this one - haven't checked any others.
![image](https://user-images.githubusercontent.com/4991075/39333228-b231bd48-496f-11e8-93ac-fed24a25b142.png) 
@Myoldmopar Do we really want to start moving to these functions for units? Current docs only have a few instances of this style in the airflownetwork section of the I/O Ref.  I guess I could get used to the negative exponent notation but I find it less readable.

Reply:
The units was an error in the command used, I'll fix any others in Boiler:HotWater.

The rendering style can be changed globally to show as follows:
![units](https://user-images.githubusercontent.com/20280470/39352111-f5cd47f4-4a03-11e8-9d6c-dbe8ac9159d6.png)
rather than the negative exponent format. @Myoldmopar could you confirm which is preferred?

It is also possible to set up macros for specific unit combinations, e.g. ```\volumeFlowRate``` could be defined to produce m<sup>3</sup>/s so that it will be consistent throughout the docs and can be changed in a single location if required.

Reply:
The macros would be nice. I guess you know my vote on preferred rendering style. Please add a tip about this in the wiki [Documentation page.](https://github.com/NREL/EnergyPlus/wiki/Documentation)